package.json

```diff
@@ -16,26 +16,26 @@
   },
   "author": "mabl",
   "dependencies": {
-    "@actions/core": "^1.2.7",
-    "@actions/github": "^4.0.0",
-    "async-retry": "^1.2.3",
-    "axios": "^0.21.1",
-    "cli-table3": "^0.5.1",
-    "moment": "^2.24.0"
+    "@actions/core": "^1.6.0",
```

Comment:
Pin and update everything.

Reply:
The dependencies should be pinned to a specific version right? Asking due to the `^`.

Reply:
You are right. Let me fix this.